com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -2595,6 +2595,9 @@ void Callback(CommandBuffer c, HDCamera cam)
             // We need to make sure the viewport is correctly set for the editor rendering. It might have been changed by debug overlay rendering just before.
             cmd.SetViewport(hdCamera.finalViewport);
 
+            if (camera.cameraType == CameraType.SceneView)
```

Comment:
The test should check if we are the current texturewire mode no ?

var sv = UnityEditor.SceneView.sceneViews[i] as UnityEditor.SceneView;
sv.cameraMode.drawMode != UnityEditor.DrawCameraMode.TexturedWire 

(but not sure how to get the current scene view as there is multiple one).

Reply:
Yeah looks like scene view render mode of camera is not exposed to scripts, so checks could only be done in a very roundabout way. `RenderWireOverlay` internally does nothing if the mode is not TexturedWire.

Reply:
ok, good to know